src/main/java/jp/kusumotolab/kgenprog/ga/crossover/Crossover.java

```diff
@@ -130,6 +130,35 @@ public Crossover initialize(final Random random,
         return new UniformCrossover(random, firstVariantSelectionStrategy,
             secondVariantSelectionStrategy, generatingCount, requiredSolutions);
       }
+    },
+
+    /**
+     * 直列交叉を表す型．
+     *
+     * @see jp.kusumotolab.kgenprog.ga.crossover.CascadeCrossover
+     */
+    Cascade {
+      /**
+       * 交叉を行うインスタンスを生成するメソッド．
+       *
+       * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
+       *      SecondVariantSelectionStrategy, int)
```

Comment:
`initialize`の引数にintが足りないです．
最後にintを足していただけないでしょうか．
```suggestion
       * @see Crossover.Type#initialize(Random, FirstVariantSelectionStrategy,
       *      SecondVariantSelectionStrategy, int, int)
```

`Random`，`SinglePoint`，`Uniform`にも同様の変更をいただけると助かります．